.changeset/good-hairs-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/react-renderer': patch
```

Comment:
This should be `minor` instead of `patch`!

Reply:
Sure!